x/mongo/driver/topology/connection.go

```diff
@@ -374,6 +374,17 @@ func (c *connection) readWireMessage(ctx context.Context, dst []byte) ([]byte, e
 		}
 	}
 
+	// In the case of an isMaster response where MaxMessageSize has not yet been set, use a hard-coded
+	// 48000000 bytes instead.
+	maxMessageSize := c.desc.MaxMessageSize
+	if maxMessageSize == 0 {
+		maxMessageSize = 48000000
```

Comment:
nit: Declare this as a constant at the top of the file and use it by name here instead of declaring the number inline.

Reply:
Declared as `maxMessageSizeDefault`.